### ISSUE TITLE
Warnings from React

### ISSUE THREAD

**Issue by DrummerHead** [projectstorm/react-diagrams#79]
Just copying some warnings I got from React 15.6.1 after using the library:

```
Warning: Accessing PropTypes via the main React package is deprecated, and will be removed in  React v16.0. Use the latest available v15.* prop-types package from npm instead. For info on usage, compatibility, migration and more, see https://fb.me/prop-types-docs
```

```
Warning: Accessing createClass via the main React package is deprecated, and will be removed in React v16.0. Use a plain JavaScript class instead. If you're not yet ready to migrate, create-react-class v15.* is available on npm as a temporary, drop-in replacement. For more info see https://fb.me/react-create-class
```

```
Warning: Accessing factories like React.DOM.path has been deprecated and will be removed in v16.0+. Use the react-dom-factories package instead.  Version 1.0 provides a drop-in replacement. For more info, see https://fb.me/react-dom-factories
```

Cheers!

**Comment by dylanvorster** [comment]
hey man, thanks for this. Although I recently upgraded react (are you using 2.5 >.> ?)

**Comment by DrummerHead** [comment]
Yup! `storm-react-diagrams@^2.5.0` from my `yarn.lock` :)

From [storm-react-diagrams' `yarn.lock`](https://github.com/projectstorm/react-diagrams/blob/5ed6a3995f42d1ed57e322efc315056108810794/yarn.lock#L4671-L4679) I can see you're also using React 15.6.1, so you ought to see these warnings on the browser console.

They come from a file called `lowPriorityWarning.js`, so I'm guessing they're something to be worried about for future versions of React :)

**Comment by dylanvorster** [comment]
well the reason I'm asking, is because I've moved the entire codebase to TSX (or so I thought?)
Im also not using create class anywhere (or so I also thought) x_x Are you sure the createClass is not in your project files?

**Comment by dylanvorster** [comment]
see: https://github.com/projectstorm/react-diagrams/search?q=createClass&type=Code&utf8=✓

**Comment by dylanvorster** [comment]
Mostly fixed in caadbf4a9a0b03bde4ee32c5f1b9789ee7ed34bb

**Comment by dylanvorster** [comment]
Fixed in 2.5.2
